### PR TITLE
Add createBlessedRenderer function

### LINKED ISSUE #79: Allow other blessed apps
Given blessed is... somewhat unmaintained, it would be nice to be able to pass in a different function to be used (ie - only require blessed if none is passed). That way we could use forked versions with updated code.

Thoughts?

### PR DESCRIPTION
**Pull request by d4rky-pl**
Fixes #79 

The diff looks long mostly because of whitespace changes, there are no major modifications to the code.

### CODE REVIEW

**Comment by Yomguithereal** [comment]
Thanks @d4rky-pl. I will need some time to read this. Just discovered `neo-blessed`. Didn't know about this.

**Comment by d4rky-pl** [comment]
@Yomguithereal someone linked it either in issues here or in original `blessed`. The diff looks complicated but it's basically moving everything below `type Instance` into a function so `blessed` is passed from outside and not from `require` and then returning what used to be the old `render` function.

If it helps I can split it into two commits, one without adding indentation (which is what destroyed github diff view) and then second that fixes indentation. Just let me know!

File: src/fiber/fiber.js [hunk 1]
```diff
@@ -232,9 +232,11 @@ const createBlessedRenderer = function(blessed) {
   }
 }
 
-const blessed = require('blessed');
-
 module.exports = {
-  render: createBlessedRenderer(blessed),
+  render: function render(element, screen, callback) {
+    const blessed = require('blessed');
```
Comment (Yomguithereal) [inline src/fiber/fiber.js]: Can this dynamic import make it more difficult for some bundlers?

**Comment by Yomguithereal** [comment]
Everything looks good to me. Can you add some docs about the newly created function though, please?

> I have more ideas on how to improve the codebase, builds, bundle size, APIs etc but I don't know if it makes sense to start discussions, send PRs etc.

Don't hesitate to do so. I don't have much time lately but contributions are always welcome.

File: src/fiber/fiber.js (hunk 1 above)
Comment (d4rky-pl) [reply src/fiber/fiber.js]: Not as far as I know but it's a good question and I'm gonna check first

Comment (d4rky-pl) [reply src/fiber/fiber.js]: **Tested:**
`webpack` - everything was OK
`rollup` - unable to compile without marking blessed as external dependency anyway (which keeps require('blessed')) due to blessed codebase not passing strict mode - [known issue](https://github.com/rollup/rollup-plugin-commonjs/issues/122), won't be fixed

If there are other bundlers you'd like me to test, go ahead, but as far as I know (may be wrong!) dynamic `require` is nothing out of the ordinary.

**Comment by d4rky-pl** [comment]
I've rebased the branch to fix the merge conflict and updated README

**Comment by Yomguithereal** [comment]
Thanks @d4rky-pl. I'll merge now.

**Pull request merged**